src/schema/tags.ts

```diff
@@ -76,5 +81,22 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
         hits: hits.map((x) => ({ name: x.value })),
       };
     },
+    onboardingTags: async (source, args, ctx): Promise<GQLTagSearchResults> => {
+      const hits = await ctx.getRepository(Keyword).find({
+        select: ['value'],
+        where: {
+          status: 'allow',
```

Comment:
Since we set onboarding manually, this feels redundant. Especially when we indexed onboarding flag by itself

Reply:
Added value and status index in 6cbb60a0b26ac82fce03c61396a8064a65c17443